### DISCUSSION TITLE
Where is the Community

### DISCUSSION THREAD

**Discussion by Levinethan** [yomorun/yomo#146] [General] [answered]
Is there have any chatroom to talk about more deeply usage of yomo like QQgroup or WeChatgroup？

**Comment by fanweixiao** [comment, marked as answer]
YoMo's Discord group: https://discord.gg/CTH3wv9
(Invitation linked updated at Nov 8, 2022)

**Reply by zhangli2946** [reply]
Man, link is expired.

**Reply by fanweixiao** [reply]
> Man, link is expired.

Thanks, I've updated the link, and set to never expires.
